static/js/emoji_picker.js

```diff
@@ -38,14 +46,7 @@ function get_all_emoji_categories() {
 }
 
 function get_frequently_used_emojis() {
```

Comment:
We can now remove this function and use `exports.frequently_used_emojis_list` directly, correct?

Reply:
Yeah alright I'll fix this